packages/ckeditor5-font/tests/fontsize/fontsizeediting.js

```diff
@@ -121,6 +121,48 @@ describe( 'FontSizeEditing', () => {
 
 					expect( editor.getData() ).to.equal( '<p>f<span style="font-size:18px;">o</span>o</p>' );
 				} );
+
+				it( 'should convert <font size=".."> styling', () => {
```

Comment:
Maybe an additional test for stripping `<font size>` when `supportAllValues` is not set would be nice.

Reply:
Sure, just added.